python/oneflow/nn/modules/unique.py

```diff
@@ -16,14 +16,17 @@
 import oneflow as flow
 
 
-def unique_op(input, *, return_inverse=False, return_counts=False, dtype=flow.int):
+def unique_op(
+    input, *, sorted=True, return_inverse=False, return_counts=False, dtype=flow.int
```

Comment:
```suggestion
    input, sorted=True, return_inverse=False, return_counts=False, dtype=flow.int
```
这里不需要了，当时设置 keyword only 是因为参数还没对齐，防止用户 `flow.unique(x, True, ...)` 这样使用